timesketch/ui/core/upload.directive.ts

```diff
@@ -46,8 +44,8 @@ export const tsCoreFileModel = ['$parse', function ($parse) {
             const model = $parse(attrs.tsCoreFileModel)
             const modelSetter = model.assign
 
-            element.bind('change', function (){
-                scope.$apply(function (){
+            element.bind('change', function () {
```

Comment:
These could be changed to arrow functions.

Reply:
I would rather create a separate PR which changes it everywhere.

Reply:
Sounds good!